src/pages/docs/creators/ray/troubleshooting.md

```diff
@@ -61,9 +121,13 @@ It produces an output like this:
   71258 ?        Sl     2:35 yagna
 ```
 
-The above shows `ray-on-golem` webserver and the `yagna` daemon are running.
+The above shows the `ray-on-golem` webserver and the `yagna` daemon are running.
+
+Note that Ray on Golem leaves the `yagna` daemon running on purpose - it stays connected to the Golem network maintaining current information about the providers so that when you start up your cluster again the nodes are found quicker. 
```

Comment:
```suggestion
Note that Ray on Golem leaves the `yagna` daemon running on purpose - it stays connected to the Golem network maintaining current information about the providers so that when you start up your cluster again the nodes are found more quickly. 
```